release/release.go

```diff
@@ -377,6 +400,16 @@ func genDashboardReleaseNotes(tmpl *template.Template, _ string, rd dashboardRel
 	return b, nil
 }
 
+func genCLIReleaseNotes(tmpl *template.Template, _ string, rd cliReleaseNoteData) (*bytes.Buffer, error) {
```

Comment:
Aside from variable names, what's the significant difference between this function and the ones above? I'm starting to think this can all be handled in one function since they all seem to be copy/pastes of one another.

Reply:
I did update this code to remove the specific `gen...ReleaseNotes`, but to reach that I did create the following interface:
```golang
type releaseNote interface {
	Fill(milestone string) error
	Template() string
	Repo() string
}
```

this removes some duplicated code, let me know what you think.